data/en/ecommerce_0051.json

```diff
@@ -621,7 +679,8 @@
                                     },
                                     {
                                         "list": [
-                                            "Portable computers or other portable devices such as mobile phones or smart phones, laptops, tablets or similar portable devices."
+                                            "Portable computers or other portable devices such as mobile phones or smart phones, laptops, tablets or similar portable devices.",
```

Comment:
AFAICS the standard practice is to avoid punctuation at the end of guidance. We should at least be consistent with this.